pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -149,22 +151,22 @@ public boolean isFinal() {
   public class SegmentBuildDescriptor {
     final File _segmentTarFile;
     final Map<String, File> _metadataFileMap;
-    final StreamPartitionMsgOffset _offset;
+    final Checkpoint _offset;
```

Comment:
Why rename this to Checkpoint?

Reply:
`StreamPartitionMsgOffset` now extends `Checkpoint` so base class was kept here. It is not compulsory now to implement MsgOffset